src/main.cpp

```diff
@@ -5,8 +5,8 @@
 
 int main() {
     // SETUP
-    const uint32_t renderCalls = 50;
-    const uint32_t samples = 10000;
+    const uint32_t renderCalls = 50000;
+    const uint32_t samples = 1000000;
```

Comment:
I think it would be nice to replace those static variables with command line arguments/flags and use those values only as defaults. Can you add that? Otherwise, I wouldn't set the commited values so high.